sections/queueing_model/waiting_time/main.tex

```diff
@@ -0,0 +1,104 @@
+    W^{(1)} = \frac{\sum_{\substack{(u,v) \, \in S_A^{(1)} \\ v \geq C}} 
+    \frac{1}{C \mu} \times (v-C+1) \times \pi(u,v)}{\sum_{(u,v) \, 
+    \in S_A^{(1)}} \pi(u,v)}
+\end{equation}
+
+The mean waiting time of type 2 individuals:
+
+\begin{equation}\label{eq:closed_form_waiting_type_2}
+    W^{(2)} = \frac{\sum_{\substack{(u,v) \, \in S_A^{(2)} \\ min(v,T) \geq C}} 
+    \frac{1}{C \mu} \times (\min(v+1,T)-C) \times \pi(u,v)}{\sum_{(u,v) \, 
+    \in S_A^{(2)}} \pi(u,v)}
+\end{equation} 
+
+The overall mean waiting time:
+
+\begin{equation}\label{eq:overall_waiting_time}
+    W = \frac{\lambda_1 P_{L'_1}}{\lambda_2 P_{L'_2} + \lambda_1 P_{L'_1}} W^{(1)} 
+    + \frac{\lambda_2 P_{L'_2}}{\lambda_2 P_{L'_2} + \lambda_1 P_{L'_1}} W^{(2)}
+\end{equation}
+ 
+Here \(S_A^{(1)}\) and \(S_A^{(2)}\) are the set of accepting states for type
+1 and type 2 individuals. These are the set of states that the model is able
+to accept a specific type of individuals.
+
+\begin{equation}\label{eq:accepting_states_type_1}
+    S_A^{(1)} = \{(u, v) \in S \; | \; v < N \}
+\end{equation}
+
+\begin{equation}\label{eq:accepting_states_type_2}
+    S_A^{(2)}=
+    \begin{cases}
+        \{(u, v) \in S \; | \; u < M \}, & \textbf{if } T \leq N\\
+        \{(u, v) \in S \; | \; v < N \}, & \textbf{otherwise}
+    \end{cases}
+\end{equation}
+
+Equation \ref{eq:overall_waiting_time} makes use of the proportion of type 1 
+and type 2 individuals that are not lost to the system. These probabilities are 
+given by \(P_{L'_1}\) and \(P_{L'_2}\) where:
+
+\begin{equation}\label{eq:proportion_of_accepting_individuals}
+    P_{L'_1} = \sum_{(u,v) \, \in S_A^{(1)}} \pi(u,v) \hspace{2cm}
+    P_{L'_2} = \sum_{(u,v) \, \in S_A^{(2)}} \pi(u,v)
+\end{equation}
+ 
+Appendix \ref{sec:appendix_mean_waiting} gives more details on the recursive
+formula that equations (\ref{eq:closed_form_waiting_type_1}),
+(\ref{eq:closed_form_waiting_type_2}) and (\ref{eq:overall_waiting_time})
+originate from. 
+
+\subsubsection{Example}
+Figures \ref{fig:markov_vs_des_waiting_time_comparison_1},
+\ref{fig:markov_vs_des_waiting_time_comparison_2} and
+\ref{fig:markov_vs_des_waiting_time_comparison_overall} show a comparison 
+between the calculated mean waiting time using Markov chains and the simulated 
+waiting time using discrete event simulation over a range of values of 
+\(\lambda_2\) (details of the discrete event simulation model are given in 
+appendix~\ref{sec:appendix_des}).
+The simulation was ran 100 times and the recorded mean waiting time at each 
+iteration is used to populate the violin plots.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=.8\textwidth]{imgs/waiting_time_comparison/waiting_1.pdf}
+    \caption{
+        Comparison of mean waiting time for type 1 individuals between values 
+        obtained from the Markov chain formulas and values obtained from 
+        simulation.
+    }
+    \label{fig:markov_vs_des_waiting_time_comparison_1}
+\end{figure}
```

Comment:
Let us add 2 other versions of these.

- [ ] Include a table with 3 columns. The rows correspond to the parameters of the system (choose 3 relatively different systems).
- [ ] Plot the waiting time comparisons as 3 separate pdfs. This will lead to 3 subfigures for each figure. (3 plots next to each other)
- [ ] Include a bit more of a discussion in the captions (confirming obvious general trends as well as the fact that they match).